pom.xml

```diff
@@ -50,7 +50,7 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <transformation.api.version>0.2.0</transformation.api.version>
+        <transformation.api.version>0.3.0-SNAPSHOT</transformation.api.version>
```

Comment:
❗️ We should not be dependent on a snapshot of the API.

Reply:
This was done in order to get the API changes. See https://github.com/connexta/ion-transformation-api/blob/master/pom.xml#L35

Reply:
then we need to release that repo since all clients (including CDR) should rely on a release version.

Reply:
This will be addressed in a separate ticket.